src/main/resources/static/js/book.js

```diff
@@ -0,0 +1,96 @@
+function myFunc1() {
+    var survey;
+
+    survey = {
+        name : "survey1",
+        questions : [ { type: "openEnded", question: "q1" }, { type: "openEnded", question: "q2" } ]
+    };
+
+    $.ajax({
+         method: "POST",
+         url   : "http://localhost:8080/createSurvey",
```

Comment:
We should make the URL flexible, as in have the "localhost:8080" portion defined as a constant elsewhere. I can change this when I implement React.

Reply:
Ya, it wouldn't be a bad idea to maybe have a constant or properties file.